Source/tmsg.cpp

```diff
@@ -38,6 +38,8 @@ void tmsg_add(byte *pbMsg, uint8_t bLen)
 	TMsg **tail;
 
 	TMsg *msg = static_cast<TMsg *>(std::malloc(bLen + sizeof(*msg)));
+	if (msg == nullptr)
+		throw std::bad_alloc();

```

Comment:
Would prefer not to introduce exceptions to the code base, maybe we can use `app_fatal` instead?

Reply:
Ok, `app_fatal` it is. What message would be appropriate?

`app_fatal("Failed to allocate memory")`?

Reply:
Sounds good 🙂